email/sender.go

```diff
@@ -0,0 +1,103 @@
+package email
+
+import (
+	"context"
+	"time"
+
+	"github.com/NebulousLabs/skynet-accounts/database"
+
+	"github.com/sirupsen/logrus"
+	"gitlab.com/NebulousLabs/errors"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+const (
+	// sleepBetweenScans defines how long the sender should sleep between its
+	// sweeps of the DB.
+	sleepBetweenScans = 30 * time.Second
+)
+
+var (
+	// ServerDomain holds the name of the name of this particular server.
+	ServerDomain = "siasky.net"
```

Comment:
Can we reference the value in the env file for consistency?

Reply:
It's already there but I haven't mentioned it in the docstring. I guess I added it to the env file later.
Updating the docsting.